packages/e/echarts.json

```diff
@@ -15,7 +16,19 @@
       {
         "basePath": "dist",
         "files": [
-          "**/*"
+          "**/*.js?(.map)"
+        ]
+      },
+      {
+        "basePath": "i18n",
+        "files": [
+          "*.js"
+        ]
+      },
+      {
+        "basePath": "theme",
+        "files": [
+          "**/*.js"
         ]
       }
```

Comment:
These are pulling the files from `i18n/*.js` to `*.js` without the subdirectory -- do you want to preserve the subdirectory?

Reply:
Not necessary. `i18n` has commonly no subdirectory. Thanks.

Reply:
The same goes for theme, all the files from `theme` are being pulled up into the root directory -- I'm concerned there might be file collision and it'll be confusing for folks?

Reply:
Could we make two directories `i18n` and `theme` in the root directory? It may be confusing indeed if they are all in the root directory.

Reply:
```suggestion
      {
        "basePath": "",
        "files": [
          "i18n/*.js",
          "theme/**/*.js"
        ]
      }
```

Reply:
Thank you for patient help. Is there anything else that needs to be concerned and corrected?